### DISCUSSION TITLE
Can't open FetchXML Builder

### DISCUSSION THREAD

**Discussion by dbvendere** [rappen/FetchXMLBuilder#1005] [Q&A]
In XrmToolBox, after FetchXML Builder installation, I receive this error when I try to open it:

An error occured when trying to display this tool: Impossible de charger le type 'AppInsights' à partir de l'assembly 'XrmToolBox, Version=1.2023.9.66, Culture=neutral,PublicKeyToken=null'.

**Comment by rappen** [comment]
Please make sure you have latest version of XrmToolBox.
See similar earlier issues we had: https://jonasr.app/missing-xrmtoolbox-pluginsstore/
Should be solved by latest version. See release notes https://fetchxmlbuilder.com/releases/1-2023-11/
